cfgov/unprocessed/js/modules/nav-primary.js

```diff
@@ -5,85 +5,98 @@
-    if ( es.get.isThisExpanded( $primaryNav ) ) {
-      es.set.toggleExpandedState( $primaryNav, 'false', function() {
-        $primaryTrigger.focus();
-      } );
-    } else {
-      es.set.toggleExpandedState( $primaryNav, 'true', function() {
-        $primaryLink.first().focus();
-      } );
-    }
-
-    es.set.toggleExpandedState( $primaryTrigger );
-    es.set.toggleExpandedState( $( 'body' ) );
-  } );
+  _primaryNav = document.querySelector( '.js-primary-nav' );
+  _primaryTrigger = document.querySelector( '.js-primary-nav_trigger' );
+  _primaryLinks = _primaryNav.querySelectorAll( '.js-primary-nav_link' );
+  _subNavs = _primaryNav.querySelector( '.js-sub-nav' );
 
-  $primaryLink.on( 'click', function( event ) {
-    event.preventDefault();
-
-    var $this = $( this );
-    var $thisSubNav = $this.siblings( '.js-sub-nav' );
-    var $otherSubNavs = $subNavs.not( $thisSubNav );
-    var firstLink = $thisSubNav.find( 'a' ).first();
-
-    if ( es.get.isOneExpanded( $otherSubNavs ) ) {
-      es.set.toggleExpandedState( $primaryLink, 'false' );
-      es.set.toggleExpandedState(
-        $otherSubNavs,
-        'false',
-        function() {
-          es.set.toggleExpandedState( $thisSubNav );
-          firstLink.focus();
-        }
-      );
-    } else if ( es.get.isThisExpanded( $thisSubNav ) ) {
-      es.set.toggleExpandedState( $thisSubNav, 'false', function() {
-        $this.focus();
-      } );
-    } else {
-      es.set.toggleExpandedState( $thisSubNav, null, function() {
-        firstLink.focus();
-      } );
-    }
+  _primaryTrigger.addEventListener( 'click', _primaryTriggerClicked );
 
-    es.set.toggleExpandedState( $this );
-  } );
+  for ( var i = 0, len = _primaryLinks.length; i < len; i++ ) {
+    _primaryLinks[i].addEventListener( 'click', _primaryLinkClicked );
+  }
 
-  $subBack.on( 'click', function() {
-    var $thisSubNav = $( this ).closest( '.js-sub-nav' );
-    var $thisPrimaryLink = $( this )
-                           .closest( '.js-primary-nav_item' )
-                           .find( '.js-primary-nav_link' );
-    es.set.toggleExpandedState( $primaryLink, 'false' );
-    es.set.toggleExpandedState( $thisSubNav, 'false', function() {
-      $thisPrimaryLink.focus();
-    } );
-  } );
+  var subBack = _primaryNav.querySelector( '.js-sub-nav_back' );
+  subBack.addEventListener( 'click', _subBackBtnClicked );
 
 /*
   TODO: wrap in a breakpoint handler along with other hover
   events to open the menu after #907 is merged
 
-  $primaryNav.on( 'mouseleave', function() {
-    es.set.toggleExpandedState( $subNavs, false );
+  _primaryNav.addEventListener( 'mouseleave', function() {
+    expandedState.toggleExpandedState( subNavs, false );
   } );
 */
 }
 
+function _subBackBtnClicked( event ) {
+  var target = event.currentTarget;
+  var thisSubNav = domTraverse.closest( target, 'js-sub-nav' );
+  var thisPrimaryLink = domTraverse.closest( target, 'js-primary-nav_item' );
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/2ca2f703bc99e6d7be15f01bcec782a31f1b6039